testsuite/standalone/src/test/java/org/jboss/as/test/integration/management/cli/ModuleTestCase.java

```diff
@@ -186,6 +186,20 @@ public void addRemoveModuleMetaInf() throws Exception {
         testModuleAndSlotExist(false, slot, getModulePath());
     }
 
+    @Test
+    public void addModuleWithWhitespacesInDependencies() throws Exception {
+        // should trim the surrounding whitespaces
+        try {
+            cli.sendLine("module add --name=" + MODULE_NAME
+                    + " --resources=" + jarFile.getAbsolutePath()
+                    + " --dependencies=[ org.jboss.logging ]"
```

Comment:
I commented above but don't have any particular opinion about using the brackets in this test, so no argument from me about removing them. It's good to have coverage somewhere that the [xxx,yyy] syntax parses correctly, but that very well may be covered elsewhere and even if it isn't a test specifically aimed at that syntax seems better.